.travis.yml

```diff
@@ -7,8 +7,13 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - nightly
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
Yes, it fails ;)
And lot of projects slowly drop support for it (including Symfony which is in the dep tree)